Submitter:	Timothy Legge <timlegge@gmail.com>
Date:		2020-04-04
Specify rpath in Makefile.PL to avoid redundant rpath

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -57,6 +57,9 @@
 	#MAN3PODS        => {},         # Pods will be built by installman. 
 	INC		=> "-I$INC_DIR",
 	VERSION_FROM	=> 'BerkeleyDB.pm',
+	dynamic_lib => {
+		OTHERLDFLAGS => "-Wl,-rpath,/usr/lib/perl5/core_perl/CORE"
+	},
 	XSPROTOARG	=> '-noprototypes',
 	DEFINE		=> "$OS2 $WALL $TRACE",
 	#'macro'		=> { INSTALLDIRS => 'perl' },
